ethpm_types/contract_type.py

```diff
@@ -204,6 +205,44 @@ def extract_sourcemap_item(expanded_row, item_idx, previous_val=None):
             yield item
 
 
+class PCMapItem(BaseModel):
+    line_start: Optional[int]
+    column_start: Optional[int]
+    line_end: Optional[int]
+    column_end: Optional[int]
+
+
+class PCMap(BaseModel):
+    """
+    As part of the source output, the compiler provides the a map of program counter values to the
+    spans of source code that
```

Comment:
I think it's missing some words

Reply:
Dang it, pushed too soon before finishing my thoughts lol

Updated in 00073a4